draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,32 +627,47 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
-Moreover, if an A or AAAA response arrives before an SVCB response, the
-client MAY wait at least CD milliseconds before connecting as if the
+## Optimistic pre-connection and connection reuse
+
+If an address response arrives before the corresponding SVCB response, the
+client MAY initiate a connection as if the
```

Comment:
Might want to just rewrap the paragraph.

Reply:
Done